UnityProject/Assets/Scripts/Util/SweetExtensions.cs

```diff
@@ -111,7 +111,7 @@ public static uint NetId(this GameObject go)
 	/// Creates garbage! Use very sparsely!
 	public static Vector3 AssumedWorldPosServer(this GameObject go)
 	{
-		return go.GetComponent<ObjectBehaviour>()?.AssumedWorldPositionServer() ?? WorldPosServer(go);
+		return go.OrNull()?.GetComponent<ObjectBehaviour>()?.AssumedWorldPositionServer() ?? WorldPosServer(go);
```

Comment:
still happens

Reply:
This will still error as `WorldPosServer()` doesnt expect null. 

And aran is right, do a null check in the case that it error'd in the log not in this function

Reply:
https://github.com/unitystation/unitystation/pull/7932/files#r765769786

This is the correct error, which is causing nulling this one. Dont need this change anymore